tests/by-util/test_shuf.rs

```diff
@@ -147,6 +149,81 @@ fn test_head_count() {
     );
 }
 
+#[test]
+fn test_zero_head_count_pipe() {
+    let result = new_ucmd!().arg("-n0").pipe_in(vec![]).succeeds();
+    // Output must be completely empty, not even a newline!
```

Comment:
The comment is redundant
```suggestion
```

Reply:
I disagree: Without the comment there's the danger that someone might think it's a bug, or unintentional.